package.json

```diff
@@ -5,8 +5,9 @@
   "license": "Apache-2.0",
   "repository": "https://github.com/Netflix/x-test",
   "type": "module",
-  "main": "x-test.js",
-  "module": "x-test.js",
+  "exports": {
```

Comment:
This replaces `main` as a way to be more explicit. In particular, it’s valuable to restrict the “accepted” import statements. In this case, we keep it the same as it was. Internal files can import whatever, but integrators will be warned if they import a file they’re not meant to.